src/pages/Reader.tsx

```diff
@@ -30,6 +30,35 @@ class Reader extends React.Component<any, State> {
 
   constructor(props: any) {
     super(props);
+    let readerSettings = {
+      readerMode: ReaderMode.longStrip,
+      imageSizing: ImageSizing.fitHeight,
+      colourTheme: ColourTheme.light,
+      menu: Menu.open,
+    }
+
+    const savedSettings = localStorage.getItem('carrotSettings');

```

Comment:
possibly overkill but can also define constants for each localStorage key

Reply:
Might be a consideration for the future. I suggest keeping everything together though, because otherwise we would have to pass every single individual setting as a prop - or the reader settings just holds the functions so its more clean

Reply:
just meant do sm like
```
export const carrotSettingsKey = 'carrotSettings';
```
and then
```
localStorage.getItem(carrotSettingsKey)
```

Reply:
meh